machines/openstack/worker.machineset.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: cluster.k8s.io/v1alpha1
+kind: MachineSet
+metadata:
+  name: worker
+  namespace: {{ .TargetNamespace }}
+  labels:
+    sigs.k8s.io/cluster-api-cluster: {{.OpenStack.ClusterName}}
+    sigs.k8s.io/cluster-api-machine-role: worker
+    sigs.k8s.io/cluster-api-machine-type: worker
+spec:
+  replicas: {{.OpenStack.Replicas}}
+  selector:
+    matchLabels:
+      sigs.k8s.io/cluster-api-machineset: worker
+      sigs.k8s.io/cluster-api-cluster: {{.OpenStack.ClusterName}}
+  template:
+    metadata:
+      labels:
+        sigs.k8s.io/cluster-api-machineset: worker
+        sigs.k8s.io/cluster-api-cluster: {{.OpenStack.ClusterName}}
+        sigs.k8s.io/cluster-api-machine-role: worker
+        sigs.k8s.io/cluster-api-machine-type: worker
+    spec:
+      providerConfig:
```

Comment:
this is specific for the aws actuator you'll need to update this config to match the definition on the actuator https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/cloud/openstack/openstackproviderconfig/v1alpha1/types.go#L24

Reply:
A-ha, gotcha! Will do